zone/merc.cpp

```diff
@@ -5575,20 +5575,17 @@ Merc* Client::GetMerc() {
 	return (tmp);
 }
 
-uint8 Client::GetNumberOfMercenaries() {
-
-	uint8 numMercs = 0;
+uint8 Client::GetNumberOfMercenaries()
+{
```

Comment:
Moved bracket?

Reply:
We prefer newline for function definitions from what I’ve seen in modern code. Inconsistent over the years.